### DISCUSSION TITLE
WordPress Core Integrations

### DISCUSSION THREAD

**Discussion by zackkrida** [WordPress/openverse#22] [Ideas]
We intend to integrate Openverse directly with WordPress, to provide WordPress users easy access to openly licensed media, starting with Creative Commons licensed images and soon expanding to audio.

Here are a few initial ideas for integrations:

- [ ] Add Openverse as an external media source to the Image block (see Jetpack: https://github.com/Automattic/jetpack/tree/ba2dd6c30be137baad96aafb2466d0aed7e182e5/projects/plugins/jetpack/extensions/shared/external-media)
	- Would this mean moving the external media functionality from Jetpack to core/gutenberg directly? 
	- Would Pexels remain, or only Openverse initially?
- [ ] A feature plugin that adds license and attribution info as first-class meta fields to items in the media library
    - Is this a good idea?
    - Is this necessary?
- [ ] Match some functionality of an existing plugin https://github.com/lenasterg/ls-wp-ccsearch

**Comment by fcoveram** [comment]
The image block seems more aligned with the direction we want to push. More and new people using openly licensed content without expert knowledge about licenses and contributions. Then, the core integration, visible just by typing /img and see the Openverse option, is a seamless experience.

Regarding Pexels, I would have it as an option since it's a different use case. I saw that some photos do not require any attribution. Quote from its [License](https://www.pexels.com/license/) page.
>  Attribution is not required. Giving credit to the photographer or Pexels is not necessary but always appreciated.

The plugin requires extra steps for the installation and, therefore, we will need to work on a comms strategy to attract users. Although for current ones that might not be a problem.

**Comment by zackkrida** [comment]
This discussion will be conducted elsewhere and doesn't need to be preserved. Openverse _is_ being added to Jetpack and WP.com's _External Media_ feature.
